eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -477,145 +460,145 @@ object Commitments {
+          val commitments1 = addLocalProposal(fail)
           Right((commitments1, fail))
-        case None => Left(UnknownHtlcId(commitments.channelId, cmd.id))
+        case None => Left(UnknownHtlcId(channelId, cmd.id))
       }
     }
   }
 
-  def receiveFail(commitments: Commitments, fail: UpdateFailHtlc): Either[ChannelException, (Commitments, Origin, UpdateAddHtlc)] =
-    commitments.getOutgoingHtlcCrossSigned(fail.id) match {
-      case Some(htlc) => commitments.originChannels.get(fail.id) match {
-        case Some(origin) => Right(addRemoteProposal(commitments, fail), origin, htlc)
-        case None => Left(UnknownHtlcId(commitments.channelId, fail.id))
+  def receiveFail(fail: UpdateFailHtlc): Either[ChannelException, (Commitments, Origin, UpdateAddHtlc)] =
+    getOutgoingHtlcCrossSigned(fail.id) match {
+      case Some(htlc) => originChannels.get(fail.id) match {
+        case Some(origin) => Right(addRemoteProposal(fail), origin, htlc)
+        case None => Left(UnknownHtlcId(channelId, fail.id))
       }
-      case None => Left(UnknownHtlcId(commitments.channelId, fail.id))
+      case None => Left(UnknownHtlcId(channelId, fail.id))
     }
 
-  def receiveFailMalformed(commitments: Commitments, fail: UpdateFailMalformedHtlc): Either[ChannelException, (Commitments, Origin, UpdateAddHtlc)] = {
+  def receiveFailMalformed(fail: UpdateFailMalformedHtlc): Either[ChannelException, (Commitments, Origin, UpdateAddHtlc)] = {
     // A receiving node MUST fail the channel if the BADONION bit in failure_code is not set for update_fail_malformed_htlc.
     if ((fail.failureCode & FailureMessageCodecs.BADONION) == 0) {
-      Left(InvalidFailureCode(commitments.channelId))
+      Left(InvalidFailureCode(channelId))
     } else {
-      commitments.getOutgoingHtlcCrossSigned(fail.id) match {
-        case Some(htlc) => commitments.originChannels.get(fail.id) match {
-          case Some(origin) => Right(addRemoteProposal(commitments, fail), origin, htlc)
-          case None => Left(UnknownHtlcId(commitments.channelId, fail.id))
+      getOutgoingHtlcCrossSigned(fail.id) match {
+        case Some(htlc) => originChannels.get(fail.id) match {
+          case Some(origin) => Right(addRemoteProposal(fail), origin, htlc)
+          case None => Left(UnknownHtlcId(channelId, fail.id))
         }
-        case None => Left(UnknownHtlcId(commitments.channelId, fail.id))
+        case None => Left(UnknownHtlcId(channelId, fail.id))
       }
     }
   }
 
-  def sendFee(commitments: Commitments, cmd: CMD_UPDATE_FEE, feeConf: OnChainFeeConf): Either[ChannelException, (Commitments, UpdateFee)] = {
-    if (!commitments.localParams.isInitiator) {
-      Left(NonInitiatorCannotSendUpdateFee(commitments.channelId))
+  def sendFee(cmd: CMD_UPDATE_FEE, feeConf: OnChainFeeConf): Either[ChannelException, (Commitments, UpdateFee)] = {
+    if (!this.localParams.isInitiator) {
```

Comment:
nit: why did you start using `this.` in this function and below, and above you directly accessed fields like `localParams` without the `this` prefix? I guess you simply replaced `commitments.` with `this.`?

Reply:
I flip-flopped between the two and intended to leave a comment asking for reviewer's opinion. `this` is a bit more fool-proof, but it's verbose and the compiler should warn us in case we accidentally shadow a variable's name.

Reply:
I don't have a strong preference either, but it would be nice to have consistency across the file.
It's a bit easier to make sure things are consistent when *not* using `this.`: we simply have to check its absence.
It's harder to verify that `this.` is used for every field, the compiler doesn't tell us and we have no automated way of checking it.
So I weakly favor not using `this.`, but it's not clear-cut...

Reply:
Done in [9e2ee70](https://github.com/ACINQ/eclair/pull/2539/commits/9e2ee70d5c326a8c8fab929daeea6dcfcf397757).